### PR TITLE
Nb/vision/map output + doc

### PR DESCRIPTION
**Pull request by Nadav-Barak**
small updates in output table + doc

### CODE REVIEW

File: docs/source/checks/vision/performance/plot_mean_average_precision_report.py [hunk 1]
```diff
@@ -22,49 +22,43 @@
...
+# %%
 # Define a condition 
 # ==================
-# We can define a condition that checks whether our model's average precision score is not less than
-# a given threshold
+# We can define a condition that checks whether our model's mean average precision score is not less than
+# a given threshold for all bounding box sizes.
 
-check = MeanAveragePrecisionReport().add_condition_test_average_precision_not_less_than(0.4)
+check = MeanAveragePrecisionReport().add_condition_test_mean_average_precision_not_less_than(0.4)
```
Comment (ItayGabbay) [inline docs/source/checks/vision/performance/plot_mean_average_precision_report.py:62]: Did you also change the condition in the check itself?

Comment (Nadav-Barak) [reply docs/source/checks/vision/performance/plot_mean_average_precision_report.py]: No, just the condition we are displaying to the user in the doc.
The other one is less informative.

**Review by ItayGabbay — APPROVED** [review]

File: deepchecks/vision/checks/performance/mean_average_precision_report.py [hunk 2]
```diff
@@ -31,7 +31,7 @@
 
 
 class MeanAveragePrecisionReport(SingleDatasetCheck):
-    """Summarize mean average precision metrics on a dataset and model per IoU and area range.
+    """Summarize mean average precision metrics on a dataset and model per IoU and bounding box area size.
```
Comment (noamzbr) [inline deepchecks/vision/checks/performance/mean_average_precision_report.py:34]:
```suggestion
    """Summarize mean average precision metrics on a dataset and model per IoU and bounding box area.
```

File: docs/source/checks/vision/performance/plot_mean_average_precision_report.py [hunk 3]
```diff
@@ -22,49 +22,43 @@
...
 result
 
-#%%
+# %%
 # Observe the check’s output
 # --------------------------
-# The result value is a dataframe that has the average precision score per each area range and IoU.
+# The result value is a dataframe that has the Mean Average Precision score for different bounding box area sizes.
+# We report the mAP for different IoU thresholds: 0.5, 0.75 and an average of mAP values
+# for IoU thresholds between 0.5 and 0.9 (with jump size of 0.05).
```
Comment (noamzbr) [inline docs/source/checks/vision/performance/plot_mean_average_precision_report.py:52]:
```suggestion
# for IoU thresholds between 0.5 and 0.9 (with a jump size of 0.05).
```

File: docs/source/checks/vision/performance/plot_mean_average_precision_report.py [hunk 4]
```diff
@@ -22,49 +22,43 @@
...
 
 result.value
 
-#%%
+# %%
```
Comment (noamzbr) [inline docs/source/checks/vision/performance/plot_mean_average_precision_report.py:56]:
```suggestion
#%%
```

**Review by noamzbr — APPROVED** [review]

**Pull request merged**